desktop/extensions.md

```diff
@@ -86,8 +87,7 @@ Feedback can be given to an extension author through a dedicated Slack channel o
 extension, select:
     - Support
     - Slack
-    - Issues
-    You'll be sent to a page outside of Docker Desktop to submit your feedback.
+    - Issues. You'll be sent to a page outside of Docker Desktop to submit your feedback.
 
 If an extension does not provide a way for you to give feedback, contact us and we'll pass on the feedback for you. 
```

Comment:
How can they contact us? Is there a link to take them to a contact form?

Reply:
Yes there is a link in DD. But good point - I have made it clearer in the doc!